### ISSUE TITLE
Documentation for how to build noresm2cmor contains an empty link

### ISSUE THREAD

**Issue by oyvindseland** [NorESMhub/noresm2cmor#363]
**Describe the bug**
The link below in the description links to a webpage that does not exist.

https://pcmdi.github.io/cmor-site/download.htm

**To Reproduce**

Try following the link

**Additional context**
Is it possible to link to an existing library build?

**Comment by YanchunHe** [comment]
@oyvindseland 

I updated the README file for building instructions, in the last commit: bd1bf3b3

If you are running it on NIRD, you don't need to install the CMOR library, but use directly the installed the copy under NS9560K.

Hope it will help.

**Comment by oyvindseland** [comment]
Will try to use the NS9560K copy. mpiifort does not exist any more on nird so the makefile can not be used.

**Comment by YanchunHe** [comment]
> Will try to use the NS9560K copy. mpiifort does not exist any more on nird so the makefile can not be used.

On NIRD, one can use 'module load Intel/2022a' for example, to load Intel compiler.

However, I noticed that this loaded Intel compiler for parallel program does work for the 'noresm2cmor' (has some strange error reading name list file, have not dig into this in detail, neither tried other Intel version. The Intel/2022a was chosen because it is close to the version that was previously used to build the CMOR libraries).

In short, currently, the GNU compiler is recommended. That is, use `Makefile_cmor3mpi.nird_gnu` or `Makefile_cmor3.nird_gnu` for now.

**Comment by YanchunHe** [comment]
Reopen if in need.
